app/src/androidTest/java/com/github/campus_capture/bootcamp/MapVoteTest.java

```diff
@@ -54,9 +59,8 @@ public class MapVoteTest {
     LatLng noLoc = new LatLng(0, 0);
 
     private static boolean hasAttacked;
-    private static boolean voteReturnCode;
 
-    final BackendInterface mock = new PlaceholderBackend() {
+    private final BackendInterface mock = new BackendInterface() {
```

Comment:
You should keep using PlaceholderBackend for tests because it provide a default implementation of all methods of BackendInterface so you can override only the methods you need.
If you use directly BackendInterface  everytime new method are added to BackendInterface you will need to implement them here even if you don't you them.